code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4364,3 +4407,64 @@
 	New()
 		..()
 		reagents.add_reagent(NUTRIMENT,3)
+
+/obj/item/weapon/reagent_containers/food/snacks/bait
+	name = "bait"
+	desc = "Don't fall for it."
+	icon_state = "bacon" //placeholder
+	bitesize = 1
+	New()
+		..()
+		reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/reagent_containers/food/snacks/fish
+	name = "raw fish"
+	desc = "The product of a real man who lives off the land."
+	icon_state = "meat" //placeholder
+	bitesize = 1
+	var/list/fish_guts = list(/obj/item/weapon/reagent_containers/food/snacks/meat/fish_fillet) // contains anything the fish will will drop when filleted
+	New()
+		..()
+		reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/reagent_containers/food/snacks/fish/attackby(obj/item/weapon/W, mob/user)
+	if(istype(W, /obj/item/weapon/kitchen/utensil/knife/large))
+		Fillet(user)
+	else
+		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/fish/proc/Fillet(mob/user)
+	if(!isturf(loc) && !user.is_holding_item(src))
+		return
+	var/ground = get_turf(src)
+	if(fish_guts.len == 1) // put fish_guts in hand if there's only 1 and we are filleting in hand
+		var/obj/item/weapon/temp = fish_guts[1]
+		var/obj/item/weapon/cont = new temp
+		if(user.is_holding_item(src))
+			if(user.drop_item(src, user))
```

Comment:
This shouldn't really be checked since it's guaranteed to be deleted anyways. I'd suggest deleting it, then doing the `put_in_hands`